src/Host.cpp

```diff
@@ -2803,10 +2803,10 @@ std::pair<bool, QString> Host::createMiniConsole(const QString& windowname, cons
         if (!pW) {
             pC->resize(width, height);
             pC->move(x, y);
-            return {false, QStringLiteral("miniconsole \"%1\" exists already. moving/resizing \"%1\".").arg(name)};
+            return {false, QStringLiteral("miniconsole \"%1\" exists already: moving/resizing \"%1\".").arg(name)};
```

Comment:
I don't understand this change. The `:` makes this confusing. Why did the `.` not suffice? There are more changes like this I don't understand.

Reply:
It's proper grammar to use `:` to extend the thought instead of starting a separate sentence - looks better on the eye.

What other ones didn't you follow?

Reply:
`"exists already"` or `"already exists"` - I can't put my finger on it but the second sounds better - especially as an introduction to what follows *after* the `:`

Reply:
I agree, latter sounds better.

Reply:
No idea how : connects the 2 sentences. I would expect information on where or how the name exists already.
I would connect the sentences with "therefore" or leave them separated or drop the second completely.